.github/workflows/pull-request.yaml

```diff
@@ -84,6 +84,10 @@ jobs:
         run: |
           cd web/
           npm test
+      - name: Run eslint
+        run: |
+          cd web/
+          npm run eslint
```

Comment:
What do you think if we first run the `eslint` and then the `unit tests`? Just to make it fail fast if something does not pass the eslint validation.